ping-meta/index.js

```diff
@@ -3,44 +3,40 @@ const github = require('@actions/github');
 const ini = require('ini');
 const toml = require('toml');
 
+function getPlainFile(octokit, repo, ref, path) {
+    return octokit.repos.getContent({
+        owner: repo.owner, // should be 'robotpy'
+        repo: repo.repo,
+        ref,
+        path,
+    }).then(result => Buffer.from(result.data.content, 'base64').toString());
+}
+
+function getPackageMetadata(octokit, repo, ref) {
+    return getPlainFile(octokit, repo, 'pyproject.toml')
+        .then(result => result.tool['robotpy-build'].metadata)
+        .catch(() => getPlainFile(octokit, repo, 'setup.cfg')
+                    .then(result => result.metadata));
+}
+
 // ref https://github.com/peter-evans/repository-dispatch/blob/master/src/main.ts
 async function run() {
+    const context = github.context;
+
+    if (!context.ref.startsWith('refs/tags/'))
+        return;
+
+    const tag = context.ref.slice('refs/tags/'.length);
+    // Skip prerelease tags.
+    if (!/^\d+(?:\.\d+)*$/.test(tag))
```

Comment:
This doesn't seem right? What is this checking for? I feel like there should be an alphabetical check somewhere..

Reply:
This checks whether the version has only numeric components. If it doesn't, then it's a prerelease and we bail.